zulip_bots/zulip_bots/pubbot.py

```diff
@@ -0,0 +1,107 @@
+
+import json
+import logging
+import time
+
+from .config import get_config
+from .zulip import Client
+
+import requests
+session = requests.Session()
+
+def stream():
+	tt = 0
```

Comment:
The one and two letter variable names are not the most illuminating for someone unfamiliar with the format of these messages.

Reply:
I am also unfamiliar with the format of these messages. I have no idea what these names stand for, I'm just using them verbatim from the API.